eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -520,7 +536,11 @@ class Router(val nodeParams: NodeParams, watcher: ActorRef, initialized: Option[
         stay
       } else {
         log.info("validating shortChannelId={}", c.shortChannelId)
-        watcher ! ValidateRequest(c)
+        Kamon.runWithContextEntry(shortChannelIdKey, c.shortChannelId) {
+          Kamon.runWithSpan(Kamon.spanBuilder("validate-channel").tag("shortChannelId", c.shortChannelId.toString).start(), finishSpan = false) {
```

Comment:
How will that span be stopped? I'm not sure it's useful to start it here since we're just sending a message to another actor, it's more useful to do all the spanning inside the watcher isn't it?

Reply:
It is stopped in the `ValidateResult` handler. The point is to be able to measure time spent in the queue, which I think is important to measure backlogs.

Reply:
Interesting, got it.